eZ/Publish/API/Repository/Tests/LocationServiceTest.php

```diff
@@ -1484,6 +1484,48 @@ public function testCopySubtree()
     }
 
     /**
+     * Test for the copySubtree() method.
+     *
+     * @see \eZ\Publish\API\Repository\LocationService::copySubtree()
+     * @depends eZ\Publish\API\Repository\Tests\LocationServiceTest::testLoadLocation
+     */
+    public function testCopySubtreeWithAliases()
+    {
+        $repository = $this->getRepository();
+        $urlAliasService = $repository->getURLAliasService();
+
+        $mediaLocationId = $this->generateId('location', 43);
+        $demoDesignLocationId = $this->generateId('location', 56);
+        /* BEGIN: Use Case */
```

Comment:
yes for now on this one.

The idea behind ` /* BEGIN: Use Case */` was to generate examples from these automatically, so we would always have tested and automatically maintained example. Reality took a different turn so in practice you can skip these until we decide to dedicate time to a second attempt at that.